<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>NuberNext</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Endfield Launcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gameLauncherPage" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>741</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QStackedWidget" name="announcementSlider">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>221</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(20, 20, 20);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>44</x>
         <y>18</y>
         <width>131</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="assets.qrc">:/assets/endfield_logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
    <widget class="QFrame" name="announcementList">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>411</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(20, 20, 20, 150);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NuberNext</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <hintingpreference>PreferNoHinting</hintingpreference>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 255, 30);</string>
      </property>
      <property name="text">
       <string>Announcements</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(20, 20, 20, 150);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>34</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NuberNext</family>
        <hintingpreference>PreferNoHinting</hintingpreference>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-left: 6px;
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Endfield Client [Beyond 0.2.58] is now available!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>62</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NuberNext</family>
        <hintingpreference>PreferNoHinting</hintingpreference>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-left: 6px;
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Endfield Client [Beyond 0.2.57] is no longer supported!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NuberNext</family>
        <hintingpreference>PreferNoHinting</hintingpreference>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-left: 6px;
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Join discord server &lt;span style=&quot; text-decoration: underline;&quot;&gt;https://discord.gg/4y2wQ5Byzh&lt;/span&gt; for more info!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="gameLaunchButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>420</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NuberNext</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
       <kerning>true</kerning>
       <hintingpreference>PreferNoHinting</hintingpreference>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(254, 241, 4);
	color: rgb(30, 30, 30);
	border: 0px;
	border-radius: 5px;
}

QPushButton:hover {
    background-color: #dcd414; /* Slightly lighter green */
}

QPushButton:pressed {
    background-color: #c3bc12; /* Slightly darker green */
}</string>
     </property>
     <property name="text">
      <string>Launch</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="assets.qrc">:/assets/endfield_logo_v2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="gameLaunchInfo">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>427</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NuberNext</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferNoHinting</hintingpreference>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Neural Network Connecting.....</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="gameLaunchProgressBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>449</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	border: 1px solid rgb(100, 100, 100);
	border-radius: 8px;
	background-color: #ffffff;
}

QProgressBar::chunk {
	background-color: #f4eb16;
	border-radius: 7px;
}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="backgroundImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/assets/launcher_bg_a.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="navbar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: rgba(25, 25, 25, 220);
}
</string>
    </property>
    <widget class="QPushButton" name="gameLauncherNavButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(254, 241, 4);
	color: rgb(30, 30, 30);
	border: 0px;
	border-radius: 5px;
	border-image: url(:/assets/game_icon.png) no-repeat center center fixed;
}

QPushButton:hover {
    background-color: #dcd414; /* Slightly lighter green */
}

QPushButton:pressed {
    background-color: #c3bc12; /* Slightly darker green */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="serverLauncherNavButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NuberNext</family>
       <pointsize>16</pointsize>
       <bold>true</bold>
       <hintingpreference>PreferNoHinting</hintingpreference>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(254, 241, 4);
	color: rgb(30, 30, 30);
	border: 0px;
	border-radius: 5px;
}

QPushButton:hover {
    background-color: #dcd414;
}

QPushButton:pressed {
    background-color: #c3bc12; /* Slightly darker green */
}</string>
     </property>
     <property name="text">
      <string>PS</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="gradientEffect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>801</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background: qlineargradient(
        x1: 0, y1: 0,
		x2: 0, y2: 1,
        stop: 0 rgba(30, 30, 30, 0),
		stop: 0.2 rgba(30, 30, 30, 200),
        stop: 0.5 rgba(30, 30, 30, 255)
    );
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="serverLauncherPage" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>741</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QStackedWidget" name="announcementSlider_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>221</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(20, 20, 20);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_5">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>32</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="assets.qrc">:/assets/github_logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>48</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>NuberNext</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
         <hintingpreference>PreferNoHinting</hintingpreference>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ArkField PS</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>71</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>NuberNext</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <hintingpreference>PreferNoHinting</hintingpreference>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>@ SuikoAkari</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6"/>
    </widget>
    <widget class="QFrame" name="announcementList_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>411</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(20, 20, 20, 150);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NuberNext</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
        <hintingpreference>PreferNoHinting</hintingpreference>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 255, 30);</string>
      </property>
      <property name="text">
       <string>Announcements</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(20, 20, 20, 150);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>34</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NuberNext</family>
        <hintingpreference>PreferNoHinting</hintingpreference>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-left: 6px;
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>DotNet 8.0 is Required to run the server program!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>62</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NuberNext</family>
        <hintingpreference>PreferNoHinting</hintingpreference>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
margin-left: 6px;
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>MongoDB is Required to run the server program!</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="serverLaunchButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>420</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NuberNext</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
       <kerning>true</kerning>
       <hintingpreference>PreferNoHinting</hintingpreference>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(254, 241, 4);
	color: rgb(30, 30, 30);
	border: 0px;
	border-radius: 5px;
}

QPushButton:hover {
    background-color: #dcd414; /* Slightly lighter green */
}

QPushButton:pressed {
    background-color: #c3bc12; /* Slightly darker green */
}</string>
     </property>
     <property name="text">
      <string>Launch</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>251</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="assets.qrc">:/assets/endfield_logo_v2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="serverLaunchInfo">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>427</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NuberNext</family>
       <pointsize>8</pointsize>
       <bold>false</bold>
       <hintingpreference>PreferNoHinting</hintingpreference>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Neural Network Connecting.....</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="serverLaunchProgressBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>449</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	border: 1px solid rgb(100, 100, 100);
	border-radius: 8px;
	background-color: #ffffff;
}

QProgressBar::chunk {
	background-color: #f4eb16;
	border-radius: 7px;
}</string>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>126</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>NuberNext</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
       <hintingpreference>PreferNoHinting</hintingpreference>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>| SERVER by SuikoAkari</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="titleBar" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>707</x>
       <y>2</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	color: rgb(30, 30, 30);
	border: 0px;
	border-image: url(:/assets/close_icon.png) no-repeat center center fixed;
	margin: 3px;
}

QPushButton:hover {
    background-color: rgba(20, 20, 20, 50); /* Slightly lighter green */
}

QPushButton:pressed {
    background-color: rgba(20, 20, 20, 100); /* Slightly darker green */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minimizeButton">
     <property name="geometry">
      <rect>
       <x>672</x>
       <y>3</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgba(0,0,0,0);
	color: rgb(30, 30, 30);
	border: 0px;
	border-image: url(:/assets/minimize_icon.png) no-repeat center center fixed;
	margin: 3px;
}

QPushButton:hover {
    background-color: rgba(20, 20, 20, 50); /* Slightly lighter green */
}

QPushButton:pressed {
    background-color: rgba(20, 20, 20, 100); /* Slightly darker green */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>backgroundImage</zorder>
   <zorder>gradientEffect</zorder>
   <zorder>gameLauncherPage</zorder>
   <zorder>navbar</zorder>
   <zorder>serverLauncherPage</zorder>
   <zorder>titleBar</zorder>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
  <include location="assets.qrc"/>
  <include location="assets.qrc"/>
  <include location="assets.qrc"/>
  <include location="assets.qrc"/>
  <include location="assets.qrc"/>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
